### DISCUSSION TITLE
2.6.8 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#2688] [讨论]
· Mod 扩展信息中 Forge 等单词强制将首字母大写（#2360，@WTP016-CN）
· 不再自动设置 forceUnicodeFont（#2547，@LeComte909）
· 当使用 Java Wrapper 安装失败时，会禁用 Java Wrapper 并重试
· 反转右下角圆形进度条的高亮颜色，以符合直觉
· 更新 8 月赞助者列表，感谢大家的支持！
· 崩溃分析优化
　　支持分析缺失 MixinBootstrap 导致的崩溃（#2649，by @zkitefly）
　　支持分析更多 Java 版本错误导致的崩溃（#2649，by @zkitefly）
　　支持分析 Java 虚拟机的堆栈信息
　　支持从 debug.log 分析 Forge Mod 文件名
· 帮助库更新
　　添加所有自 PCL 2.6.5 以来追加的新自定义特性的教学
　　添加使用 Intellij Idea 编辑帮助库的教学（PCL2Help#323，by @Ian-Rin）
　　更新 MyCard 等控件的自定义属性教学（PCL2Help#324，@IambicCave）
　　为多人模式被禁用添加新的解决方案（PCL2Help#325，by @WForst-Breeze）
　　完善 Mod 安装教程（PCL2Help#326，by @WForst-Breeze）
　　移除部分不适用的属性教学（#2510，@IambicCave）
· 修复了以下 26 个 Bug
　　特定 Mod 的名称显示有误（#1567，@Chiloven945）
　　管理员权限下无法拖拽文件（#2127，@Dino-doge）
　　有多个任务时总下载进度显示可能有误（#2138，@youzi-2333）
　　使用 Zulu Java 可能导致安装失败（#2232，@Xx114514xX1819）
　　拖拽无效文件时会复制到临时文件夹（#2325，@Element49）
　　不兼容 Your Options Shall Be Respected Mod（#2385，@ShenXuan233）
　　个别情况下无法安装特定 Modrinth 整合包（#2390，@wjsw3369）
　　特定操作后会错误显示更多页面的左边栏（#2416，@Xiaolv114518）
　　无法为部分特殊版本安装 Fabric / Fabric API（#2418，@TanChenyu）
　　远古版本 c0.31 20100130 的 Wiki 链接有误（#2418，@TanChenyu）
　　特定操作顺序下左边栏的项目可能不会显示（#2436，@HaNWorkspace）
　　Mod 搜索会搜到 Modrinth 插件（#2458，@Chiloven945）
　　重装的同名版本会使用删除前的版本设置（#2495，@iPodtouch6557）
　　错误地尝试在没有音乐时播放音乐导致报错（#2517，@SALTWOOD）
　　Tooltip 不在页面滚动后关闭（#2552，@Roser7419）
　　特定情况下清理垃圾时出错（#2591，@kndxhz）
　　{cave} 替换标记中换行变为空格（#2608，@iPodtouch6557）
　　某条回声洞中换行符有误（#2628，@MCSteve123）
　　删除文件夹时可能出现意外的报错（#2631，@kipper3832）
　　不允许使用 JDK 19+ 启动 Forge 1.19+（#2643，@Minecraft22H2）
　　{hint} 替换标记未使用 hints.txt 的内容（#2650，@lyzzs10）
　　背景图片、音乐可能扫描到 Thumbs.db（#2664，@hiberfile-hiberfile）
　　Mixin 崩溃分析正则有误（#2680，@g-mph）
　　无法安装特定整合包（#2681，@LSTNnnn）
　　自定义帮助中无法使用 {hint} 和 {cave} 替换标记
　　源代码中 Clip 方法的实现有误

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.8'>2.6.8</a>.</em>

**Comment by kjjjjjjjjjjjj** [comment]
I wanna be join with this project please join me I give you a Java for it 8 and 17 for 32 and 64 bit

**Reply by MoYuan-CN** [reply]
what?

**Reply by kjjjjjjjjjjjj** [reply]
I wanna be join in project

**Reply by zkitefly** [reply]
You can directly modify the code of the warehouse and send a pull request

And PCL2 is not made in Java

**Reply by kjjjjjjjjjjjj** [reply]
😣😣😣😣
